src/display/svg.js

```diff
@@ -1316,39 +1315,18 @@ SVGGraphics = (function SVGGraphicsClosure() {
       // For the construction of the SVG image we are only interested in the
       // root group, so we expose it as the entry point of the SVG image for
       // the other code in this class.
-      this.svg = rootGroup;
+      this.group = rootGroup;
 
       return svg;
     },
 
     /**
      * @private
      */
-    _ensureClipGroup: function SVGGraphics_ensureClipGroup() {
-      if (!this.current.clipGroup) {
-        var clipGroup = this.svgFactory.createElement('svg:g');
-        clipGroup.setAttributeNS(null, 'clip-path',
-                                 this.current.activeClipUrl);
-        this.svg.appendChild(clipGroup);
-        this.current.clipGroup = clipGroup;
-      }
-      return this.current.clipGroup;
-    },
-
-    /**
-     * @private
-     */
-    _ensureTransformGroup: function SVGGraphics_ensureTransformGroup() {
-      if (!this.tgrp) {
-        this.tgrp = this.svgFactory.createElement('svg:g');
-        this.tgrp.setAttributeNS(null, 'transform', pm(this.transformMatrix));
-        if (this.current.activeClipUrl) {
-          this._ensureClipGroup().appendChild(this.tgrp);
-        } else {
-          this.svg.appendChild(this.tgrp);
-        }
-      }
-      return this.tgrp;
+    _clipnAppend: function SVGGraphics_clipnAppend(element) {
```

Comment:
This name is a bit unclear. Perhaps rename this to `_appendToGroup`? The clipping seems more like a detail here.